hip-config.cmake.in

```diff
@@ -205,13 +205,11 @@ if(HIP_COMPILER STREQUAL "clang")
     endif()
   endif()
 
+  add_library(hip::clang_rt_builtins INTERFACE)
+  target_link_libraries(hip::clang_rt_builtins PUBLIC -L${HIP_CLANG_INCLUDE_PATH}/../lib/linux -lclang_rt.builtins-x86_64)
```

Comment:
This should use `find_library` to find clang_rt_builtins:

```cmake
find_library(CLANG_RT_BUILTINS lang_rt.builtins-x86_64 HINTS ${HIP_CLANG_INCLUDE_PATH}/../lib/linux PATHS /opt/rocm/llvm)
```

This way a user doesnt necessarily need to set `HIP_CXX_COMPILER` as it will still be found along the `CMAKE_PREFIX_PATH`.

Reply:
Yeah, much better. Let me do that tomorrow.

Reply:
Hm, unfortunately it's not really working @pfultz2, in the sense that it's only reasonable to assume the user / package manager passes the install prefix of LLVM, not the path to clang, and in that case the following does not work:

```
find_library(CLANG_RT_BUILTINS clang_rt.builtins-x86_64 PATH_SUFFIXES "lib/clang/*/lib/linux")
```

where `*` is an attempt to enable globbing (it's an unknown version number).

Reply:
Doesn't look like it.